src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -635,6 +635,7 @@
   "EpisodeRequested": "Episode Requested",
   "EpisodeSearchResultsLoadError": "Unable to load results for this episode search. Try again later",
   "EpisodeTitle": "Episode Title",
+  "EpisodeTitleFootNote": "Truncate episode titles with `:MAX_LENGTH` (bytes). Ellipsis (`...`) is added. Positive length truncates from the end. (e.g. `{Episode Title:30}`). Negative length truncates from the beginning. (e.g. `{Episode Title:-30}`).",
```

Comment:
I agree it's hard to cut it down too much, but I think the examples and `:MAX_LENGTH` end up being redundant, but maybe something like this.

```
Optionally control truncation to a maximum number of bytes including ellipsis (`...`). Truncating from the end (e.g. `{Episode Title:30}`) or the beginning (e.g. `{Episode Title:-30}`) are both supported.
```

For episode title it will also be automatically truncated to fit maximum file system lengths, so for this also add: `Episode titles will be automatically truncated to file system limitations if necessary`.